src/Main.java

```diff
@@ -1,36 +1,33 @@
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Stream;
 
 public class Main {
-
     public static void main(String[] args) {
-        // 第3回
-        // List
-        List<String> alcoholArray = new ArrayList<>();
-        alcoholArray.add("ビール");
-        alcoholArray.add("ワイン");
-        alcoholArray.add("日本酒");
-        alcoholArray.add("焼酎");
-        alcoholArray.add("酎ハイ");
-        alcoholArray.add("カクテル");
-        System.out.println(alcoholArray.toString());
-        for (String alcohol : alcoholArray) {
-            System.out.println(alcohol);
-        }
+        //        6文字以上の関東エリアの県名を表示する
+        List<String> areaList = Arrays.asList(
+                "Tokyo",
+                "Chiba",
+                "Saitama",
+                "Kanagawa",
+                "Gunma",
+                "Tochigi",
+                "Ibaraki",
+                "Yamanashi"
+        );
+        Stream<String> stream = areaList.stream();
+        Stream<String> stream2 = stream.filter(str -> str.length() >= 6);
+        stream2.forEach(str -> System.out.println(str));
 
-        // 例外処理
-        String userAge = "hogehoge";
-        try {
-            int age = Integer.parseInt(userAge);
-            if (age >= 20) {
-                System.out.println("You can get Alcohol!!");
-            } else {
-                System.out.println("You are too young to get Alcohol!!");
-            }
-        } catch (NumberFormatException e) {
-            System.out.println("INPUT DATA IS NOT NUMBER!!");
-            throw e;
-        }
+        //        Streamを連結する書き方
+        List<String> fruits = Arrays.asList("Orange", "Apple", "Lemon", "Grape", "Strawberry");
```

Comment:
こんな感じでひとまとめにして記述することも可能です〜
```suggestion
  Arrays.asList("Tokyo", "Chiba", "Saitama", "Kanagawa", "Gunma", "Tochigi", "Ibaraki", "Yamanashi")
                .stream()
                .filter(s -> s.length() > 5)
                .forEach(System.out::println);
```